restful-web-services/2.1/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Jakarta RESTful Web Services Specification 2.1"
+date: 2019-07-12T14:29:05-0700
+summary: "First release for Jakarta EE 8"
+---
+Jakarta RESTful Web Services provides a foundational API to develop web services 
+following the Representational State Transfer (REST) architectural pattern.
+
+* [Jakarta RESTful Web Services 2.1 Specification Document](restful-web-services_2.1.pdf) (PDF)
+* [Jakarta RESTful Web Services 2.1 Specification Document](restful-web-services_2.1.html) (HTML)
+* [Jakarta RESTful Web Services 2.1 Javadoc](./apidocs)
+* [Jakarta RESTful Web Services 2.1 TCK](https://jenkins.eclipse.org/jaxrs/job/TCK_build/8/artifact/CTS/release/JAXRS_BUILD/latest/jaxrstck-2.1.0_16-Jul-2019.zip)
+* Maven coordinates
+  * [jakarta.ws.rs:jakarta.ws.rs-api:2.1.6](https://oss.sonatype.org/content/groups/staging/jakarta/ws/rs/jakarta.ws.rs-api/2.1.6/)
+
+# Ballots
+
+## Release Review
+
+|                       |  Yes    | No      | Abstain  |
+|-----------------------|---------|---------|----------|
+|Fujitsu                |         |         |          |
+|IBM                    |         |         |          |
+|Oracle                 |         |         |          |
+|Payara                 |         |         |          |
+|Red Hat                |         |         |          |
+|Tomitribe              |         |         |          |
+|EE4J PMC               |         |         |          |
+|Participant Members    |         |         |          |
+|Committer Members      |         |         |          |
+
+# Compatible Implementations
+
+* Eclipse Jakarta RESTful Web Services 2.1.6
```

Comment:
This needs to be the name and a link to a complete Compatible Implementation,
which presumably would be some version of Jersey.